nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/Book.java

```diff
@@ -85,7 +85,7 @@ public PublishingHouse getPublisher() {
             return Collections.emptyList();
         }
         List<String> validIsbns = isbnList.stream()
-            .map(isbn -> isbn.replaceAll(ANYTHING_BUT_DIGITS_REGEX, ""))
+            .map(isbn -> isbn.replaceAll(ANYTHING_BUT_DIGITS_AND_X_REGEX, ""))
```

Comment:
This is entirely wrong as we lose validation for ISBNs that are input manually. The correct approach here is to catch the validation error and resubmit without the ISBN if-and-only-if you really want to throw away data willy-nilly. The issue should really be fixed in the data source